src/index.html

```diff
@@ -1,12 +1,335 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <title>Catalog</title>
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo"><img src="./images/logo.png" alt="logo"></a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list-item"><a href="#" class="is-active header__link">Apple</a></li>
+          <li class="header__list-item"><a href="#" class="header__link">Samsung</a></li>
+          <li class="header__list-item"><a href="#" class="header__link">Smartphones</a></li>
+          <li class="header__list-item">
+            <a href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >Laptops & Computers</a>
+          </li>
+          <li class="header__list-item"><a href="#" class="header__link">Gadgets</a></li>
+          <li class="header__list-item"><a href="#" class="header__link">Tablets</a></li>
+          <li class="header__list-item"><a href="#" class="header__link">Photo</a></li>
+          <li class="header__list-item"><a href="#" class="header__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__card" data-qa="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="cardImage"
+          class="main__card-image"
+        >
+
+        <p class="main__card-name">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="main__card-code">Product code: 195434</p>
+
+        <div class="main__card-grade">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="main__card-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="main__card-price">
+          <p class="main__card-price-text">Price:</p>
+
+          <p class="main__card-price-amount">$2,199</p>
+        </div>
+
+        <button
+          class="main__card-buy-btn"
+          data-qa="card-hover"
+        >
+        Buy
+        </button>
```

Comment:
Fix it everywhere if you have only 2 attributes
```suggestion
        <button class="card__button" data-qa="card-hover">
          Buy
        </button>
```